keps/prod-readiness/sig-apps/3335.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 3335
 alpha:
   approver: "@wojtek-t"
+beta:
+  approver: "@wojtek-t"
```

Comment:
Queued.

Can you please ensure that the KEP is added to the tracking board for 1.27 (i.e. the the corresponding issue is marked as "lead-opted-in".
[It helps with balancing the load across PRR approvers]

Reply:
Yes, upstream issue has been tracked: https://github.com/kubernetes/enhancements/issues/3335#issuecomment-1410322355